aplib.net-demo/Assets/Scripts/Doors/TempItemID.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Temporary class that gives an ID to the player, change the ID in the editor of the player object
+/// </summary>
+public class TempItemID : MonoBehaviour
```

Comment:
I don't really understand why this class is temporary and/or why it is needed

Reply:
This functionality is here to test the doors as the keys are not yet in. The reason why it's a class is so that it's clearer that it's supposed to be replaced, as opposed to adding a testing variable in another class which could be missed later on